<launch>
    <param name="/controller/type" value="MPC2"/>
    <param name="/controller/pose_topic" value="/pf/viz/inferred_pose"/>
    <node pkg="lab2" name="controller" type="main.py"/>
</launch>
